src/Pages/Home.tsx

```diff
@@ -0,0 +1,49 @@
+import "./home.css";
+import { quizOne } from "../Data/quizData";
+import { Link } from "react-router-dom";
+import { useQuiz } from "../Context/quizContext";
+import Fitness from "../Images/Fitness.png";
+import HealthyMeal from "../Images/HealthyMeal.png";
+import { NavBar } from "./NavBar";
+import {useStyles} from "../theme";
+import Box from '@material-ui/core/Box';
+import Typography from '@material-ui/core/Typography';
+import Button from '@material-ui/core/Button';
+import Image from "material-ui-image";
+
+export default function Home() {
+  const { quizDispatch } = useQuiz();
+  const classes = useStyles(); 
+
+  return (
+    <Box className = {classes.homePage}>
+      <NavBar />
+      <Box className={classes.intro}>
+        Are you a fitness fanatic?
+        <br />
+        Checkout the below quizes and have fun!
+      </Box>
+      <Box className={classes.quizChoices}>
+        {quizOne.map((genre) => (
+          <Box className={classes.choice}>
```

Comment:
Panchami this could be taken out as a component something like `QuizCard.jsx` from line 28 to 44